src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the https://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Net.Http.Json
+{
+    public static partial class HttpClientJsonExtensions
+    {
+        public static System.Threading.Tasks.Task<object?> GetFromJsonAsync(this System.Net.Http.HttpClient client, string? requestUri, System.Type type, System.Text.Json.JsonSerializerOptions? options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Now that we have one signature that doesn't take options:
```cs
GetFromJsonAsync(this HttpClient client, 
  string? requestUri, 
  Type type, 
  CancellationToken cancellationToken = default)
```

And a signature that does take options:
```cs
GetFromJsonAsync(this HttpClient client, 
  string? requestUri, 
  Type type, 
  JsonSerializerOptions? options = null, 
  CancellationToken cancellationToken = default)
```

When you try to write `client.GetFromJsonAsync(uri, typeof(Person))` below code you get the error
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0121	The call is ambiguous between the following methods or properties: 'HttpClientJsonExtensions.GetFromJsonAsync(HttpClient, Uri?, Type, JsonSerializerOptions?, CancellationToken)' and 'HttpClientJsonExtensions.GetFromJsonAsync(HttpClient, Uri?, Type, CancellationToken)'	System.Net.Http.Json.Functional.Tests (net472), System.Net.Http.Json.Functional.Tests (netcoreapp5.0)	C:\repos\runtime\src\libraries\System.Net.Http.Json\tests\FunctionalTests\HttpContentJsonExtensionsTests.cs	85	Active
```

I don't think that is something that was expected since the whole purpose of adding the non-JsonSerializerOptions overloads was the convenience to not have to name the parameter. now you have to name them every time to disambiguate.

@stephentoub @terrajobst

Reply:
The JsonSerializerOptions should not have a default value.

Reply:
https://github.com/dotnet/runtime/pull/33459#discussion_r391196530

Reply:
I'm not sure what that link is meant to highlight.  I'm saying change:
```C#
GetFromJsonAsync(this HttpClient client, 
  string? requestUri, 
  Type type, 
  JsonSerializerOptions? options = null, 
  CancellationToken cancellationToken = default)
```
to:
```C#
GetFromJsonAsync(this HttpClient client, 
  string? requestUri, 
  Type type, 
  JsonSerializerOptions? options, 
  CancellationToken cancellationToken = default)
```
Doesn't that solve the problem?

Reply:
Disregard the link, it was just complementing my first comment.

Reply:
Yep, that fixes the problem. Thanks.

Reply:
@terrajobst, this makes sense, right?